mmv1/third_party/terraform/services/sql/resource_sql_database_instance_internal_test.go

```diff
@@ -36,3 +36,40 @@ func TestMaintenanceVersionDiffSuppress(t *testing.T) {
 		})
 	}
 }
+
+func TestDatabaseVersionDiffSuppress(t *testing.T) {
```

Comment:
This needs additional tests for at least the following cases (shouldn't suppress unless stated otherwise):

- Upgrading from 8_0 non-auto-upgrade version to 8_4
- Upgrading from 8_0 non-auto-upgrade version to 8_0 auto-upgrade version
- Upgrading from 5.x to 8.4
- Upgrading from 8.0 auto-upgrade version to a different 8.0 auto-upgrade version (should suppress)
- Downgrading from 8.0 auto-upgrade version to a different 8.0 auto-upgrade version (should suppress)
- Downgrading from 8.4 to 8.0 auto-upgrade version
- Downgrading from 8.4 to 8.0 non-auto-upgrade version
- Downgrading from 8.4 to 5.x
- Downgrading from 8.0 auto-upgrade version to 8.0 non-auto-upgrade version

Reply:
Thanks!